docs/data_structures/transactions.md

```diff
@@ -24,4 +24,27 @@ data sources.
 | DKB  | Checking account, webpage  | Buchungstag, Wertstellung, Buchungstext, Auftraggeber / Begünstigter, Verwendungszweck, Kontonummer, BLZ, Betrag (EUR), Gläubiger-ID, Mandatsreferenz, Kundenreferenz |
 | DKB  | Credit card, webpage       | Umsatz abgerechnet und nicht im Saldo enthalten, Wertstellung, Belegdatum, Beschreibung, Betrag (EUR), Ursprünglicher Betrag  |
 | Consorsbank  | Checking account, webpage | Buchung, Valuta, Sender / Empfänger, IBAN / Konto-Nr., BIC / BLZ, Buchungstext, Verwendungszweck, Betrag in EUR |
-| MoneyMoney  | Banking application | Datum, Wertstellung, Kategorie, Name, Verwendungszweck, Konto, Bank, Betrag, Währung  |
\ No newline at end of file
+| MoneyMoney  | Banking application | Datum, Wertstellung, Kategorie, Name, Verwendungszweck, Konto, Bank, Betrag, Währung  |
+
+## Columns
+
+These are the columns contained in the main data structure
+of the transactions.
+
+| Column | Description |
+|--------|-------------|
+| `time` | Time when the transaction happened, format: unix timestamp |
+| `sender` | String containing the name of the sender of the transaction |
+| `sender_account` | String containing the account information of the sender |
+| `receiver` | String containing the name of the receiver of the transaction |
+| `receiver_account` | String containing the account information of the receiver |
+| `text` | Text of the transaction |
+| `amount` | Amount of money that has been transferred |
+| `currency` | Currency of the transaction like "EUR" or "USD" |
+
```

Comment:
we would need an additional column with "category" and/or "tags". Instead of listing the categories/tags, category/tag_id could be used

Reply:
Oh - I've forgotten them. Added `category` and `tags` in 41813107311ddcfa5c58d2dda3a64ccf92788247.

I set the type of both `category` and `tags` to string, rather than choosing an ID as you proposed. The reason for this is that if we had IDs, we'd have to join another table/data source in order to get the name of the tags/categories. I think that the amount of data we'll be dealing with is small enough so that we do not need IDs here. I may be wrong, but I think that it'll be easier in the beginning at least.

And if it should become necessary to add IDs later, we can do so. What do you think?

Reply:
yes, agreed